mail_edit/static/src/js/mail_edit.js

```diff
@@ -39,6 +40,19 @@ openerp.mail_edit = function (instance) {
                     },
                 });
             });
+        },
+
+        on_message_delete: function (e) {
+            var mail_message_model = new openerp.Model('mail.message');
+            var self = this;
+            mail_message_model.call('unlink', [this.id]).then(
```

Comment:
it seems to me checking https://github.com/OCA/OCB/blob/8.0/addons/mail/static/src/js/mail.js#L171 should be enough to do the right thing here already?

Reply:
why don't you call https://github.com/OCA/OCB/blob/8.0/addons/mail/static/src/js/mail.js#L1036 anyways? you get a confirmation dialog and if I read the code correctly fancy fadeout for free, then your also don't need the awkward parent assumption (what if the chatter is somewhere else on the form?)